src/tests/multihost/basic/test_basic.py

```diff
@@ -117,3 +117,29 @@ def test_ssh_login_kcm(self, multihost, enable_kcm):
         else:
             assert True
             ssh.close()
+
+    def test_case_senitivity_sudo_responder(self, multihost,
```

Comment:
I'm very happy that you added a test, but wouldn't it be better to add a new test (test_sudo.py) instead of adding new test cases to the basic test?

Reply:
okay will do that.